src/customComponents/components/Checkbox.jsx

```diff
@@ -28,13 +28,13 @@ const CheckboxWrapper = class extends Component {
   };
 
   render() {
-    const component = this.props.component;
+    const { component, translate } = this.props;
     return (
       <Checkbox
         checkboxRef={this.props.checkboxRef}
         aria-describedby={`${component.key}-error`}
-        label={component.label}
-        onChange={(event) => this.setValue(!!this.state.value ? null : "ja")}
+        label={translate(component.label)}
+        onChange={(event) => this.setValue(this.state.value === "on" ? "off" : "on")}
```

Comment:
Check if this is merged to master before, and decide which code is correct

Reply:
It was merge conflict and our code is outdated. Fixed.